docs/release-notes/log-policies.rst

```diff
@@ -0,0 +1,12 @@
+:orphan:
+
+**New Features**
+
+-  Experiments: Add a new experiment config option ``log_policies`` to allow configuring policies to take after a regex is matched.This can also be configured on a cluster or per resource pool option through task container defaults. Please see :ref:`experiment-config-reference` and :ref:`master-config-reference` for more information.
+
+   There are two action types a trial can be configured to take
+
+   - ``exclude_node``: If a trial fails and restarts, the trial will not schedule on a node that reported a log that matched the regex provided. This can be used to allow trials to avoid being scheduled on nodes with certain hardware issues like uncorrectable gpu ECC errors.
```

Comment:
...will not schedule, for its restarts, on a node that reported...

Reply:
This can be used to allow trials to avoid being retried on nodes with certain...